VimeoNetworking/VimeoNetworking/VimeoClient.swift

```diff
@@ -55,7 +55,10 @@ final public class VimeoClient
     }
     
         /// Dictionary containing URL parameters for a request
-    public typealias RequestParameters = [String: AnyObject]
+    public typealias RequestDictionary = [String: AnyObject]
```

Comment:
Agreed. Gonna change these guys to "RequestParametersDictionary" and "RequestParametersArray"